eks-anywhere-common/Addons/Partner/Aqua/external-secret.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: aqua-secretstore-eksa
+  namespace: aqua
+spec:
+  refreshInterval: 1m
+  secretStoreRef:
+    name: eksa-secret-store
+    kind: ClusterSecretStore
+  target:
+    name: aqua-secrets-from-ps          # Specify the name for the Kubernetes Secret
+  data:
+    - secretKey: aqua-enforcer-token    # Key in Kubernetes Secret
+      remoteRef:
+        key: /aqua-enforcer/enforcer-token  # Key in AWS Parameter Store
+
+    - secretKey: aqua-image-username
+      remoteRef:
+        key: /aqua-enforcer/username
```

Comment:
We have already received the access token. What should be the username, password and gateway values foe us to configure AWS Secrets Manager secret?

Reply:
Based on our discussion today, im good with this secrets part, resolving this comment.